src/index.js

```diff
@@ -1,34 +1,87 @@
 import './style.css';
+import {
+  createTask,
+  saveItemToLocalStorage,
+  getItemFromLocalStorage,
+  deleteCompletedTasks,
+  deleteTask,
+} from './activity.js';
 
-const toDo = [
-  {
-    description: 'Wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Take a walk',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Go to the movie club with friends',
-    completed: false,
-    index: 2,
-  },
-];
-
-const displayToDoList = (items) => {
+const displayTasks = () => {
+  const tasks = getItemFromLocalStorage();
   const toDoList = document.getElementById('to-do-list');
-  items.forEach((toDo) => {
+  toDoList.innerHTML = '';
+  tasks.forEach((task, index) => {
     const toDoItem = document.createElement('li');
-    toDoItem.className = ('todo-item-list');
-    toDoItem.innerHTML = `<input type = "checkbox" class = "checkbox"${toDo.completed ? 'checked' : ''}></input>
-    <p>${toDo.description}</p>`;
+    toDoItem.className = ('to-do-item-list');
+
+    const anotherItem = document.createElement('div');
+    anotherItem.className = 'another-item';
+
+    const taskInput = document.createElement('input');
+    taskInput.className = 'checkbox';
+    taskInput.type = 'checkbox';
+    if (task.completeted) {
+      taskInput.setAttribute('checked', '');
+    }
+
+    taskInput.onclick = (e) => {
+      if (e.target.checked) {
+        tasks[index].completed = true;
+        e.target.parentNode.children[1].classList.add('strike-through');
+      } else {
+        tasks[index].completed = false;
+        e.target.parentNode.children[1].classList.remove('strike-through');
+      }
+      saveItemToLocalStorage(tasks);
+    };
+
+    anotherItem.appendChild(taskInput);
+
+    const taskDescription = document.createElement('p');
```

Comment:
- [ Required ] Please consider adding a function for editing task descriptions as stated in the project requirements, currently, I'm not able to edit the task description. The reason for this not being possible are as follows
  - There is no function for editing the task descriptions
  - Instead of using a `p` tag, I suggest you use an `input` of type `text` element  which will make it possible to edit the task descriptions

## Project requirements
<img src="https://user-images.githubusercontent.com/16392046/221174044-90b6564d-4dd7-4cca-a047-e7462b5294f9.png" />